docs/CREATE_ACCOUNT.md

```diff
@@ -5,49 +5,46 @@ If your app presents Kin Spend and Earn offers to your users, then each user nee
 >**NOTE:** Kin Ecosystem SDK must be initialized before any interaction with the SDK, in order to do that you should call `Kin.initialize(…)` first.
 
 
+#### Login
 *To create or access a user’s Kin account:*
 
-Call `Kin.login(…)`, passing the user’s unique ID and your chosen authentication credentials (either whitelist or JWT credentials).
-You can add a `KinCallback` and get a response when the user is logged in, and you can start send transactions and get information regarding the account balance, status etc.
+Call `Kin.login(…)`, passing a JWT credentials.
+You can add a `KinCallback` and get a response when the user is logged in, and then start send transactions, get information regarding the account balance, status etc.
```

Comment:
Add a `KinCallback` and get a response when the user is logged in and then it is possible to
send transactions, get information regarding the account balance, status etc.

Reply:
what will happen if the developer will try to request native earn before account created? login only tell he, developer, that user is fully logged in to our system

Reply:
1. 👍 
2. we should check this edge case and see.